src/main/java/frc/robot/RobotContainer.java

```diff
@@ -31,9 +31,8 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
 
-  private final DriveSubsystem driveSubsystem = new DriveSubsystem();
+  private final DriveSubsystem driveSubsystem = new DriveSubsystem(true);
```

Comment:
Could make where it is the main robot an enemy for better readability, but this works fine performance-wise, so it's purely stylistic.